### ISSUE TITLE
Cannot copy from soft-serve's out (via mouse in the terminal)

### ISSUE THREAD

**Issue by curio77** [charmbracelet/soft-serve#462]
**Describe the bug**
For me, in Konsole/X11/Linux (but I'm pretty sure it's not specific), it's impossible to copy anything from soft-serve's output, in particular not the SSH clone commands.  Being able to do so is something pretty elementary in a terminal and hence a pretty major issue, IMO.  I suspect it's not a soft-serve issue but rather one of the underlying libraries, but I'm not familiar enough with the charmbracelet ecosystem to tell which.

**To Reproduce**
1. Connect to a soft-serve host via SSH from a terminal/console; in my case, it's Konsole on X11 in Linux.
2. See the list of repos, including SSH clone commands.
3. Try to select one of the clone commands (or really anything) with the mouse (drag over it, double-click it) and realize this is not possible.
4. Be disappointed because this should be a given in any terminal output. :-(

**Expected behavior**
I should be able to select text in general and the clone commands in particular, to copy and paste them.

**Screenshots**
—not helpful—

**Environment (please complete the following information):**
 - OS: Linux/Gentoo (up-to-date)
 - Terminal: Konsole (KDE/Plasma; v23.08.4)
 - Version: v0.7.4 (latest)

**Additional context**
—none—

**Comment by 20after4** [comment]
Using the `kitty` terminal, I'm able to select text by holding down shift and dragging the mouse (as well as other purely keyboard-based selection methods)

Not sure if this works in konsole but it might.

**Comment by aymanbagabas** [comment]
Hi @curio77, 
This depends on the terminal you're using, different terminals have different key bindings to select text. According to the [Konsole Handbook](https://docs.kde.org/stable5/en/konsole/konsole/konsole.pdf), "Pressing the Ctrl and Alt keys along with the left mouse button will select text in columns."

**Comment by curio77** [comment]
Indeed, Ctrl + Shift (_not_ Ctrl + Alt, though) works in Konsole.  But it's curious that that's required, never had that with anything else.

**Comment by pm4rcin** [comment]
Hi all,
so while it is possible to copy with Shift+Mouse as a console user I would like to copy things with `yy`  or similar. One project that I think did it well is Nvim btw. Maybe there should be some key to activate copy mode or should it be by default like manpages in nvim?
